draft-ietf-ppm-dap.md

```diff
@@ -867,23 +867,29 @@ with error of type 'outdatedConfig'. If the leader supports multiple HPKE
 configurations, it can use trial decryption with each configuration to determine
 if requests match a known HPKE configuration. When clients receive an 'outdatedConfig'
 error, they SHOULD invalidate any cached aggregator `HpkeConfigList` and retry with
-a freshly generated Report. If this retried report does not succeed, clients MUST
-abort and discontinue retrying.
+a freshly generated Report. If this retried report does not succeed, clients
+SHOULD abort and discontinue retrying.
+
+If a report's metadata matches that of a previously uploaded report, the leader
```

Comment:
I think this check is too stringent. For anti-replay, we require tracking the report IDs at a minimum.

Reply:
I changed the wording to discuss report ID. I think this needs the same nuance about sharding as the corresponding item in {{input-share-validation}} but I'm not sure how to effectively reference that here. We could just copy paste in the "implementation note".

Reply:
Or just point to that section?